app/communities-configs/polygon/index.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { css } from 'styled-components';
+
+import CustomSubHeader from 'components/CustomSubHeaders/CustomSubHeader';
+import CustomMobileSubHeader from 'components/CustomSubHeaders/CustomMobileSubHeader';
+
+import favicon from '!file-loader?name=[name].[ext]!images/favicon-telos.ico';
+import PolygonLogo from './images/polygon-logo.svg?inline';
+
+import RobotoRegularEOT from './fonts/Roboto-Regular.eot';
+import RobotoRegularWOFF from './fonts/Roboto-Regular.woff';
+import RobotoRegularTTF from './fonts/Roboto-Regular.ttf';
+
+const domainName = 'https://polygon.technology/';
+const domainDocs = 'https://docs.polygon.technology/';
+const domainBlog = 'https://blog.polygon.technology/';
+
+export const CustomSubHeaderConfig = {
+  design: 'telos_style',
+  styles: {
+    bg: {
+      header: 'rgba(255,255,255,1)',
+      dropdown: '#ffffff',
+    },
+    color: {
+      a: 'rgba(41,41,52,1)',
+      arrow: 'rgba(238,238,242,1)',
+    },
+    font: {
+      body: 'Roboto-Regular, Arial, sans-serif',
```

Comment:
`Roboto-Regular, Arial, sans-serif` repeats 5 times in this file, could you please create a variable for this and reuse it everywhere?